nodejs/week3/Mealsharing/src/backend/index.js

```diff
@@ -0,0 +1,13 @@
+const mealsRouter = require("./api/meals");
+const reviewsRouter = require("./api/reviews");
+const reservationsRouter = require("./api/reservations");
+const express = require("express");
+const app = express();
+app.use("/api", mealsRouter);
+app.use("/api", reviewsRouter);
+app.use("/api", reservationsRouter);
```

Comment:
I think that it'd be easier to understand which routers are responsible for which routes if you used the following instead:

```
app.use("/api/meals", mealsRouter);
app.use("/api/reviews", reviewsRouter);
app.use("/api/reservations", reservationsRouter);
```

That way you can see all the routes at a glance without having to jump to the actual router.

Reply:
That being said, I can see how the `mealsRouter` being responsible for its own full routes makes sense. Since every logic related to meals is within the same router.